microsoft-edge/webview2/how-to/remote-debugging.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Remote Debugging for WebView2 WinUI 2 (UWP) apps with Remote Tools for Microsoft Edge
+description: How to remotely debug WebView2 WinUI 2 (UWP) apps with Remote Tools for Microsoft Edge. 
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 02/23/2023
+---
+# Remote Debugging WebView2 WinUI 2 (UWP) apps with Remote Tools for Microsoft Edge
+
+Use `Remote Tools for Microsoft Edge` to enable [Microsoft Edge DevTools](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/landing/) debugging.
+
+
+<!-- ====================================================================== -->
+## Debugging Desktop WebView2 WinUI 2 (UWP) apps
+
+These steps can be used to attach `Microsoft Edge DevTools` remotely to a WebView2 WinUI 2 (UWP) app.
+Remote debugging is necessary as a current limitation to WebView2 WinUI 2 (UWP) apps is the built-in `Microsoft Edge DevTools` cannot be launched inside a store signed WebView2 WinUI 2 (UWP) app.

```

Comment:
```suggestion
Remote debugging is necessary because of a current limitation to WebView2 WinUI 2 (UWP) apps: the built-in `Microsoft Edge DevTools` cannot be launched inside a store-signed WebView2 WinUI 2 (UWP) app.
```
could probably make smoother even than this